test/cpp/api/tensor.cpp

```diff
@@ -194,6 +196,80 @@ TEST(TensorTest, ContainsCorrectValuesForManyValuesVariable) {
   ASSERT_TRUE(almost_equal(tensor[2], 3.125));
 }
 
+TEST(TensorTest, ContainsCorrectValuesForMultidimValuesVariable) {
```

Comment:
nit: long test names are hard to read. I think something like `TestMultidimTensorCtor` might get across what this is trying to communicate?